src/app/modals/login/login.component.scss

```diff
@@ -41,4 +41,10 @@
   width: 100%;
   margin-top: 25px;
 }
-
+i:after {
```

Comment:
add name to i tag and use css selecto by name to apply CSS

Reply:
added name for i tag and use selector